packages/react-strict-dom/src/types/StrictTypes.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow strict
+ */
+
+export type {
+  StrictAnimation,
+  Animatable as StrictAnimatable
+} from './StrictAnimation';
+export type { StrictElement } from './StrictElement';
+export type {
+  AnimationProgressEventTypes as StrictAnimationProgressEventTypes,
+  AnimationProgressEvent as StrictAnimationProgressEvent,
+  StrictAnimationProgressEventTarget,
+  StrictClipboardEventTarget,
+  StrictFocusEventTarget,
+  StrictInputEventTarget,
+  StrictKeyboardEventTarget,
+  StrictPointerEventTarget,
+  StrictWheelEventTarget,
+  StrictMouseEventTarget,
+  StrictTouchEventTarget,
+  StrictEventTarget
+} from './StrictEventTarget';
+export type { StrictHTMLElement } from './StrictHTMLElement';
+export type {
+  StrictHTMLInputElement,
+  StrictHTMLOptionElement,
+  StrictHTMLCollection,
```

Comment:
Probably don't want to export this

Reply:
The Element types here might actually be useful sometimes. I'm not sure if `StrictHTMLCollection` is needed though.

Generally, I erred in exporting more types than necessary to enable better types for abstractions built on top of RSD. I can remove everything other than the `*Element` types.

Reply:
Yes element types are needed, I had to use them already

Reply:
Talking about StrictHTMLCollection